README.md

```diff
@@ -0,0 +1,40 @@
+# KeyCloak Delay Authentication Flow
+
+Plugin to delay authentication of users until they get created in Keycloak and added in 3scale by RHMI operator.
+
+# System Requirements
+
+You need to have Keycloak 9.0.2 running.
```

Comment:
Now that I think of it, Keycloak is not a requirement because we use docker-compose...